.travis.yml

```diff
@@ -1,16 +1,16 @@
-sudo: false
+sudo: true
```

Comment:
Isn't it better to use `sudo: required`? https://docs.travis-ci.com/user/docker/